crates/chainspec/src/api.rs

```diff
@@ -38,7 +38,7 @@ pub trait EthChainSpec: Send + Sync + Unpin + Debug {
     fn prune_delete_limit(&self) -> usize;
 
     /// Returns a string representation of the hardforks.
-    fn display_hardforks(&self) -> impl Display;
+    fn display_hardforks(&self) -> String;
```

Comment:
let's make it return `DisplayHardforks` here instead, because we return it in all implementations